apis/proto/v1/vald/index.proto

```
@@ -26,26 +26,34 @@ option java_multiple_files = true;
 option java_outer_classname = "ValdIndex";
 option java_package = "org.vdaas.vald.api.v1.vald";
 
+// Overview
 // Represent the index manager service.
 service Index {
```

buf (1.47.2)

Service name "Index" should be suffixed with "Service".

(SERVICE_SUFFIX)